adafruit_lsm6ds.py

```diff
@@ -293,11 +294,11 @@ def acceleration(self):
 
     @property
     def gyro(self):
-        """The x, y, z angular velocity values returned in a 3-tuple and are in degrees / second"""
+        """The x, y, z angular velocity values returned in a 3-tuple and are in radians / second"""
         raw_gyro_data = self._raw_gyro_data
-        x = self._scale_gyro_data(raw_gyro_data[0])
-        y = self._scale_gyro_data(raw_gyro_data[1])
-        z = self._scale_gyro_data(raw_gyro_data[2])
+        x = self._scale_gyro_data(raw_gyro_data[0]) * _DPS_TO_RADS
```

Comment:
You could use `math.radians(self._scale_gyro_data....)` instead, and lose the _DPS_TO_RADS constant.  math.radians() is standard in Python and I checked for its presence in CircuitPython.

Reply:
@siddacious try a x, y, z = [math.radians(self._scale_gyro_data(i)) for in raw_gyro_data]

Reply:
👍